Automattic/wp-calypso
# Update blogger plan headline from "Best for hobbyists" to "Best for bloggers"

#### Changes proposed in this Pull Request

* Update blogger plan headline from "Best for hobbyists" to "Best for bloggers"
* Update blogger plan copy to reflect required updates

#### Testing instructions

* Go to /plans and plans step in the signup flow and confirm the new headline is visible
* Also confirm that the copy reflects https://docs.google.com/document/d/1qXHcPVmnBPsSr5PJbIH3PoMaa576YC8Zk5cuMIN6ePc
